docs/shader-modules/project64.md

```diff
@@ -43,7 +43,7 @@ Projected position in the clipspace.
 
 ```glsl
 void project_position_fp64(vec4 position_fp64, out vec2 out_val[2])
-void project_position_fp64(vec2 position, vec2 position64xyLow, out vec2 out_val[2])
+void project_position_fp64(vec2 position, vec2 position64Low, out vec2 out_val[2])
```

Comment:
Nit: `vec2 position64Low` -> `vec3 position64Low`

Reply:
`project_position_fp64` only deals with x and y.